.gitconfig

```diff
@@ -0,0 +1,3 @@
+[branch]
```

Comment:
Jon, we don't want this file in the shared repository. You can set your local config with the git config command ... read up on the options for that.

Reply:
Ok.  I have removed this file from my fork.